dist/api_data.py

```diff
@@ -739,6 +739,10 @@ def __ge__(self, other):
             decisions about what subsystems are throttled, and in what proportion, are made
             internally. The minimum non-zero setting is 1MB.''',
             min='0', max='1TB'),
+        Config('fsync_backgroud_period_sec', '0', r'''
+            max waiting period for fsync background, If the written threshold are not met for
+            ertain time, force to running the fysnc background.''',
```

Comment:
Can you change the wording a bit (and fix spelling mistake in the name).  I suggest:
```
Config('fsync_background_period_sec', '0', r'''
max waiting period for background fsync. If the written threshold is not met in this time, a background fsync is done.''',
```